### ISSUE TITLE
Tag text doesn't change

### ISSUE THREAD

**Issue by moshpirit** [yoyurec/logseq-awesome-styler#110]
**Info:**
- OS: Arch Linux
- Logseq ver: 0.10.5
- Plugin ver: 3.7.1
- [x] ⚠ Do i have issue with DISABLED my `custom.css`? Yes, it keeps failing.
- [x] ⚠ Do i have the same on default theme? No

**Description:**
The color of hagstags remains the same even when I try to change it.

**To Reproduce:**
 
[here's my logseq-awesome-styler.json](https://pastes.io/raw/gtzkrjeprb)

**Expected behavior:**
It uses tag text selected in options.

**Screenshots**
![](https://i.imgur.com/IVEArKJ.png)
![](https://i.imgur.com/SIS97Ld.png)

**Comment by derwok** [comment]
Yes, I can second that. My Tag-Text currently is hardly readable, and I cannot change it... would be nice if it gets fixed.

**Comment by andremann** [comment]
+1

**Comment by ffcarina** [comment]
+1 😭

**Comment by TREE37** [comment]
same 😭
